docs/UserGuide.adoc

```diff
@@ -291,46 +320,42 @@ Deletes a person with the NRIC S98654322D in ORGANice.
 // end::delete[]
 === Detecting matches: `match`
 
-Currently, ORGANice only supports kidney matching. Refer to <<FAQ>> for a better understanding of the
-procedure for kidney matching.
-
-This command displays a list of patient-donor pairs that passes both the blood type and tissue
-typing tests. +
+This command runs a kidney matching test on patients and donors in ORGANice and displays the match results.
+Refer to <<FAQ>> for a better understanding of the procedure for kidney matching.
 
-There are two types of match commands. The following list explains each type of match command and describes the result
-of executing each type of command:
-
-* To get a list of patients and their number of matches, the format is: `match ic/all`
-* To get a list of potential matches a patient has with a donor, the format is: `match ic/**NRIC**`
-
-Each match is accompanied by a success rate number,
-which indicates the degree of compatibility
-between the patient and donor. +
+You can either run a kidney matching test on all patients and donors or find matching donors for a specific patient.
+To learn how to run kidney matching tests on all patients, refer to <<Matching all patients>>. To learn how to run
+kidney matching tests on a specific patient, refer to <<Matching a specific patient>>.
 
 ==== Matching a specific patient
-For example, a user wants to match a specific patient called "John Doe" and his NRIC is "S1234568R".
+This section will explain how to run kidney matching tests on a specific patient and run you through an example.
+To run a match of a specific patient, type in a command of the format `match ic/**NRIC**`.
 
-. Type `match ic/S1234568R` in the command box and press Enter.
+Here is an example. Suppose you want to match a specific patient called "John Doe" and his NRIC is "S1234568R". This is
+what to do:
```

Comment:
Perhaps you can use "This is what **you can** do:" to make it sound more conversational.

Reply:
Amended.